packages/gui/src/components/offers/OfferManager.tsx

```diff
@@ -93,7 +101,37 @@ function OfferList(props: OfferListProps) {
     pageChange,
   } = useWalletOffers(5, 0, includeMyOffers, includeTakenOffers, 'RELEVANCE', false);
 
+  const isWalletSynced = useIsWalletSynced();
+
+  const { data: height, isLoading: isGetHeightInfoLoading } = useGetHeightInfoQuery(undefined, {
+    pollingInterval: 3000,
+  });
+  const { data: lastBlockTimeStampData, isLoading: isGetTimestampForHeightLoading } = useGetTimestampForHeightQuery({
+    height: height || 0,
+  });
+  const lastBlockTimeStamp = lastBlockTimeStampData?.timestamp || 0;
+  const currentTimeMoment = moment.unix(lastBlockTimeStamp - 20);
```

Comment:
What is the `- 20` for? Can you add a comment?

Reply:
removed the -20 part